hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -168,12 +168,39 @@ registers: [
       },
     },
 
+    { multireg: {
+        cname: "SENSOR_CTRL",
+        name: "ALERT_EN",
+        regwen: "CFG_REGWEN",
+        desc: '''
+          Each multibit value enables a corresponding alert.
+        ''',
+        count: "NumAlertEvents",
+        swaccess: "rw",
+        hwaccess: "hro",
+        compact: "false",
+        fields: [
+            { bits: "3:0",
+              name: "VAL",
+              mubi: "true"
+              desc: '''
+                kMultiBitBool4True - An alert event is enabled.
+                kMultiBitBool4False - An alert event is disabled.
+              ''',
+              resval: "False"
```

Comment:
Reset to True to prevent accidentally leaving them off? As in the tests you needed to modify, or more importantly, in firmware.

Reply:
I thought about that, but as explained below, I think this is the safer way due to the fact that these alerts can be flaky and we don't want a known bad alert pollute any of the two alert channels by default.